patcher/src/ts/redux/modules/channels.ts

```diff
@@ -55,9 +56,15 @@ export default function reducer(state: ChannelState = initialState, action: any
         selectedChannel: action.channel
       });
     case REQUEST_CHANNELS:
+      let selectedChannel:Channel;
+      if(action.selectedChannel)
```

Comment:
if (expression) statement 
or
if (expression) {
  statement;
}
are the acceptable forms.  https://github.com/airbnb/javascript#blocks--braces